examples/notebooks/healthcare/beam_nlp.ipynb

```diff
@@ -0,0 +1,642 @@
+        "\n",
+        "**Note**: This example is used from [here](https://github.com/rasalt/healthcarenlp/blob/main/nlp_public.ipynb).\n",
+        "\n",
+        "\n",
+        "\n",
+        "This example demonstrates how to set up an Apache Beam pipeline that reads a file from [Google Cloud Storage](https://https://cloud.google.com/storage), and calls the [Google Cloud Healthcare NLP API](https://cloud.google.com/healthcare-api/docs/how-tos/nlp) to extract information from unstructured data. This application can be used in contexts such as reading scanned clinical documents and extracting structure from it.\n",
+        "\n",
+        "An Apache Beam pipeline is a pipeline that reads input data, transforms that data, and writes output data. It consists of PTransforms and PCollections. A PCollection represents a distributed data set that your Beam pipeline operates on. A PTransform represents a data processing operation, or a step, in your pipeline. It takes one or more PCollections as input, performs a processing function that you provide on the elements of that PCollection, and produces zero or more output PCollection objects.\n",
+        "\n",
+        "For details about Apache Beam pipelines, including PTransforms and PCollections, visit the [Beam Programming Guide](https://beam.apache.org/documentation/programming-guide/).\n",
+        "\n",
+        "You'll be able to use this notebook to explore the data in each PCollection."
+      ],
+      "metadata": {
+        "id": "nEUAYCTx4Ijj"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "First, lets install Apache Beam."
+      ],
+      "metadata": {
+        "id": "ZLBB0PTG5CHw"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "pip install apache-beam[gcp]"
+      ],
+      "metadata": {
+        "id": "O7hq2sse8K4u"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Set the variables in the next cell based upon your project and preferences. The files referred to in this notebook nlpsample*.csv are in the format with one\n",
+        "blurb of clinical note.\n",
+        "\n",
+        "Note that below, **us-central1** is hardcoded as the location. This is because of the limited number of [locations](https://cloud.google.com/healthcare-api/docs/how-tos/nlp) the API currently supports."
+      ],
+      "metadata": {
+        "id": "D7lJqW2PRFcN"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "DATASET=\"<YOUR-DATASET-HERE>\"\n",
+        "TEMP_LOCATION=\"<YOUR-TEMP-LOCATION-HERE>\"\n",
+        "PROJECT='<YOUR-PROJECT-ID-HERE>'\n",
+        "LOCATION='us-central1'\n",
+        "URL=f'https://healthcare.googleapis.com/v1/projects/{PROJECT}/locations/{LOCATION}/services/nlp:analyzeEntities'\n",
+        "NLP_SERVICE=f'projects/{PROJECT}/locations/{LOCATION}/services/nlp'\n",
+        "GCS_BUCKET=PROJECT"
```

Comment:
We should probably follow this naming guidance for security best practice: https://cloud.google.com/storage/docs/buckets#:~:text=Don%27t%20use%20user%20IDs%2C%20email%20addresses%2C%20project%20names%2C%20project%20numbers%2C%20or%20any%20personally%20identifiable%20information%20(PII)%20in%20bucket%20names%20because%20anyone%20can%20probe%20for%20the%20existence%20of%20a%20bucket.